frameworks/python/sqlalchemy/src/run-examples-with-database-url

```diff
@@ -126,7 +127,8 @@ for filename in testfiles:
 					tables = ','.join(['`' + table[0] + '`' for table in tables])
 					cursor.execute('DROP TABLE ' + tables)
 					complete = True
-				except MySQLdb._exceptions.IntegrityError:
+				except MySQLdb._exceptions.IntegrityError as e:
+					print(e)
 					pass
```

Comment:
`pass` can get removed here since there's a function call in `print()` now

Reply:
Good point, it's been a minute since I've done any real Python ;)